packages/extensions/openapi-to-typespec/src/generate/generate-enums.ts

```diff
@@ -37,7 +39,10 @@ export function generateEnums(typespecEnum: TypespecEnum) {
             const doc = generateDocs(m);
             const kv = `"${m.name}": ${m.value}`;
             if (doc.length > 0 || !isFullCompatible) return `${doc}${kv}`;
-            else return `${generateSuppressionForDocumentRequired()}\n${kv}`;
+            else
+              return `${generateSuppressions([
+                getSuppresssionWithCode("@azure-tools/typespec-azure-core/documentation-required"),
```

Comment:
consider to declare a const string "@azure-tools/typespec-azure-core/documentation-required" to avoid the duplicates